test/support/factory.ex

```diff
@@ -166,6 +167,72 @@ defmodule Wanda.Factory do
     }
   end
 
+  def completed_with_errors(
+        %Execution{execution_id: execution_id, group_id: group_id} = execution
+      ) do
+    result = %Result{
+      execution_id: execution_id,
+      group_id: group_id,
+      check_results: [
+        %CheckResult{
+          check_id: "some_check_id",
+          expectation_results: [
+            %ExpectationResult{
```

Comment:
Do we need to hardcode all of these entries. Why don't we use other factories to populate them?